rollup.config.js

```diff
@@ -0,0 +1,63 @@
+import typescript from '@rollup/plugin-typescript';
+import json from '@rollup/plugin-json';
+import shebang from '@walrus/rollup-plugin-shebang';
+import multiInput from 'rollup-plugin-multi-input';
+import fs from "fs";
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import pkge from './package.json'
+
+let filepaths = [];
+
+const walk = (root) => {
```

Comment:
please add a global explanation on why we need this process, how it works and so all